FAIL src/json-path/json-path.test.ts

**Describe the bug**
Run `rush test:ci`

```
==[ @azure-tools/datastore ]======================================[ 2 of 19 ]==
FAIL src/json-path/json-path.test.ts
  ● Test suite failed to run

    src/json-path/json-path.ts:6:31 - error TS2307: Cannot find module '@azure-tools/codegen' or its corresponding type declarations.

    6 import { createSandbox } from "@azure-tools/codegen";
                                    ~~~~~~~~~~~~~~~~~~~~~~
```

--- a/common/changes/autorest/fix-args-tests-relability_2021-04-19-15-21.json
+++ b/common/changes/autorest/fix-args-tests-relability_2021-04-19-15-21.json
@@ -1,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "autorest",
+      "comment": "",
+      "type": "none"
+    }
+  ],
+  "packageName": "autorest",
+  "email": "tiguerin@microsoft.com"
+}

--- a/packages/apps/autorest/test/args.test.ts
+++ b/packages/apps/autorest/test/args.test.ts
@@ -44,7 +44,7 @@ describe("Args", () => {
   });
 
   it("parse args with ~ path", () => {
-    expect(parseArgs(["--configFileOrFolder:~/path/to/folder"]).configFileOrFolder).toEqual(
+    expect(join(parseArgs(["--configFileOrFolder:~/path/to/folder"]).configFileOrFolder)).toEqual(
       join(homedir(), "path/to/folder"),
     );
   });

